templates/ssh_forced_commands_wrapper.py.erb

```diff
@@ -7,14 +7,27 @@ from subprocess import call
 
 
 def check_command():
-    original_command = os.environ['SSH_ORIGINAL_COMMAND']
+    source_command = os.environ.get('zpr_rsync_cmd')
+    if not source_command:
+        sys.exit(1)
+    ssh_command = os.environ['SSH_ORIGINAL_COMMAND']
+    commands_dir = '<%= @permitted_commands %>'
```

Comment:
Templating out a python script seems awkward and makes it harder to test. Can we simply put the python script in place as a file and have it read data that we template out?